### ISSUE TITLE
Restore doesn't show apps as installed that were later installed out of band

### ISSUE THREAD

**Issue by mattfbacon** [stevesoltys/backup#817]
In connection with #815 and also generally, I have installed some apps manually that weren't automatically installed. However, I can't get Seedvault to restore the data for these apps because it doesn't notice that the apps were installed. The app id `com.example.app` gets replaced with the actual name but otherwise nothing seems to change. Even if I fully re-run the restore it still doesn't seem to notice that the apps are installed.

**Comment by grote** [comment]
The detection logic currently re-queries the installation state of the app only when you use the link we provide. So this sounds like a symptom of #816 .

**Comment by mattfbacon** [comment]
Yeah, I would believe that. I've also seen this behaviour with F-Droid where it doesn't seem to notice that the app has been installed until I do something to force a re-check. Is there no way to monitor this independently?
